### PR TITLE
Bumped API version for Dundee Plus. Exposed existing field in the CLI.

### PR DESCRIPTION
**Pull request by kc284**

### CODE REVIEW

File: ocaml/client_records/records.ml [hunk 1]
```diff
@@ -1098,6 +1098,8 @@ let host_record rpc session_id host =
 				~get:(fun () -> String.concat "; " (List.map Int64.to_string (x ()).API.host_virtual_hardware_platform_versions)) 
 				~get_set:(fun () -> List.map Int64.to_string (x ()).API.host_virtual_hardware_platform_versions) ();
 			make_field ~name:"control-domain-uuid" ~get:(fun () -> get_uuid_from_ref (x ()).API.host_control_domain) ();
+			make_field ~name:"resident-vms"
+				~get:(fun () -> String.concat "; " (List.map get_uuid_from_ref (x ()).API.host_resident_VMs)) ~expensive:true ();
```
Comment (johnelse) [inline ocaml/client_records/records.ml]: Why is this marked as `expensive`? It's just a database field.

Comment (robhoes) [inline ocaml/client_records/records.ml]: I think it needs `~get_set` also.

Comment (kc284) [inline ocaml/client_records/records.ml]: @johnelse,  it seems I don't have a proper understanding of the meaning of `expensive`; can you point me to relevant docs?

@robhoes, host.resident_VMs is DynamicRO, why does it need a ~get_set?

Comment (robhoes) [inline ocaml/client_records/records.ml]: It is "set" as in the type of a set of things, rather than a setter function.

Comment (robhoes) [inline ocaml/client_records/records.ml]: See the top of records.ml:

`		expensive: bool; (* Simply means an extra API call is required to get it *)`

**Comment by johnelse** [comment]
:shipit:

**Pull request merged**